### PR TITLE
Bug/drop down click fix

### PR DESCRIPTION
**Pull request by maidbarion**
# Description

Fixed dropdown so can select entire button

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Dev test and unit test pass

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by jchiversAND — COMMENTED** [review]
The blue bottom is showing on both the intro screens without the navigation bar, and the main screens with the bar with this fix. Whereas we need it to only show when the nav bar is there.

**Comment by jchiversAND** [comment]
The dropdown box isn't the same length as the other input fields, is shorter on both sides.

File: mobile/src/navigation/BottomStackNavigation/BottomStackNavigation.js [hunk 1]
```diff
@@ -3,17 +3,21 @@ import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
 import { FontAwesome, MaterialIcons } from '@expo/vector-icons';
 import { ActivitiesScreen, SettingsScreen, ProgressScreen } from '@screens';
 import { colours, font } from '@styles/constants';
-import { Text } from '@components/atomic';
+import { Text, SafeAreaView } from '@components/atomic';
+import { useSafeAreaInsets } from 'react-native-safe-area-context';
 
 const Tab = createBottomTabNavigator();
-
+const staticNavHeight = 80;
```
Comment (anitaschan) [inline mobile/src/navigation/BottomStackNavigation/BottomStackNavigation.js:10]: Any reason this is changed from 64 to 80?

Comment (maidbarion) [reply mobile/src/navigation/BottomStackNavigation/BottomStackNavigation.js:10]: yes, to add a bit more space for older screens - we're now calculating safe zone inset size and determining the padding from that

**Review by anitaschan — APPROVED** [review]

**Pull request merged**